src/ClientData/ProcessData.cpp

```diff
@@ -81,9 +74,13 @@ void ProcessData::UpdateModuleInfos(absl::Span<const ModuleInfo> module_infos) {
   absolute_address_to_module_in_memory_cache_.clear();
 
   for (const auto& module_info : module_infos) {
+    std::optional<orbit_client_data::ModuleIdentifier> module_id_opt =
+        module_identifier_provider_->GetModuleIdentifier(
+            {.module_path = module_info.file_path(), .build_id = module_info.build_id()});
+    ORBIT_CHECK(module_id_opt.has_value());
```

Comment:
Nit: `.value()` below is doing the same check, so it could be removed. But if you want to keep it explicit, also fine.

Reply:
Done.